Emby.Dlna/ContentDirectory/ControlHandler.cs

```diff
@@ -908,6 +908,16 @@ private QueryResult<ServerItem> GetMovieFolders(BaseItem item, User user, StubTy
                 }
             };
 
+            // If Limit is < 6 then we return too many items, so we must remove some.
```

Comment:
6?

Reply:
There are 6 default items added to the list (see code prior). 
If limit is less than 6, six are still returned.

Will change comment.